src/sources/kubernetes_logs/mod.rs

```diff
@@ -74,8 +77,12 @@ pub struct Config {
     /// Override global data_dir
     data_dir: Option<PathBuf>,
 
-    /// Specifies the field names for metadata annotation.
-    annotation_fields: pod_metadata_annotator::FieldsSpec,
+    /// Specifies the field names for Pod metadata annotation.
+    #[serde(alias = "annotation_fields")]
+    pod_annotation_fields: pod_metadata_annotator::FieldsSpec,
```

Comment:
Do we need to update the doc to use the new name?

Reply:
Ah, good call 👍